crates/pop-cli/src/commands/build/parachain.rs

```diff
@@ -26,7 +41,65 @@ impl BuildParachainCommand {
 		warning("NOTE: this may take some time...")?;
 		build_parachain(&self.path)?;
 
-		outro("Build Completed Successfully!")?;
+		success("Build completed successfully!")?;
+		let project_path = self.path.clone().unwrap_or(PathBuf::from("./"));
+		// Assumes the target directory is always `target/release`. If an option to build in debug mode is added in `build_parachain`, this will need to be modified accordingly.
+		let target_path = project_path.join("target/release");
+		// Assumes the node directory is always in `node`.
+		let node_path = project_path.join("node");
+		let binary = binary_path(&target_path, &node_path)?;
+		let mut generated_files = vec![format!("Binary generated at: \"{}\"", binary.display())];
```

Comment:
In the resulting output, this path has quotes ("") but all those for the chainspec/wasm artifacts do not. Easiest to just remove them here for consistency.